scripts/changeset-version-with-docs.ts

```diff
@@ -2,6 +2,7 @@
 
 import sh from 'shelljs';
 
+// eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
Isn't this method fully synchronous?

Reply:
Yes, we are right. I am going to just remove the `async` here